picard/metadata.py

```diff
@@ -60,7 +60,15 @@ def images_to_be_saved_to_tags(self):
         if config.setting["embed_only_one_front_image"]:
             front_image = self.get_single_front_image(images)
             if front_image:
-                return front_image
+                images = front_image
+
+        # Run coverart tag embed actions
+        try:
+            from picard.coverart import run_coverart_tag_embed_action
+            for image in images:
+                run_coverart_tag_embed_action(image)
```

Comment:
Looking at https://gist.github.com/samj1912/24a8c2d35c70285280a3921d3ef539ae and this code, i think this isn't the right way to do it, there is no processing caching.
Of course in the case of resize, since dimensions are tested, the process is done once, but it wouldn't work with, for example, a black and white conversion.

We need a way to identify the process, and see if it was applied or not on one image.